client/src/js/components/map.jsx

```diff
@@ -39,7 +39,17 @@ var Map = React.createClass({
       }
     ).addTo(map);
 
-    L.geoJson(items).addTo(map);
+    var marker = new 
+    L.Icon({
+      iconUrl: "images/marker-icon.png"
```

Comment:
it will be in the next bit that I push.
On 06/02/2015 06:05 PM, lifehackett wrote:
>
> In client/src/js/components/map.jsx 
> <https://github.com/MobilityLabs/medellin-heritage/pull/8#discussion_r31580472>:
>
> > @@ -39,7 +39,17 @@ var Map = React.createClass({
> >        }
> >      ).addTo(map);
> >
> > -    L.geoJson(items).addTo(map);
> > +    var marker = new
> > +    L.Icon({
> > +      iconUrl: "images/marker-icon.png"
>
> Gotcha. Can you add it now? Or else remove it all together.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MobilityLabs/medellin-heritage/pull/8/files#r31580472>.
>